docs/contents/community/faq.md

```diff
@@ -12,11 +12,46 @@
 
     We appreciate input and advice concerning features that are currently covered by the service broker only, for example bind/unbind or cataloging and looking forward to learn from the community how they are using service broker so that we can take this into account.
 
+## Cluster API
+
+!!! question "Question"
+    Does ACK replace [cluster api](https://github.com/kubernetes-sigs/cluster-api)?
+
+!!! quote "Answer"
+    Cluster API does a lot of really cool things and is designed to be a generic way to create Kubernetes clusters that run anywhere.
+    It makes some different design decisions with that goal in mind.
+    Some differences include:
+
+    - Cluster API is treated as your source of truth for all infrastructure.
+    This means things like the cluster autoscaler need to be configured to use cluster api instead of AWS cloud provider.
+    ACK provisions clusters using AWS as the source of truth the same way any EKS cluster does.
+    - Generic Kubernetes clusters rely on running more services in the cluster and not services from AWS.
+    Things like metrics and logging will likely need to run inside Kubernetes instead of using services like CloudWatch.
+    - IAM permission for Cluster-API Provider AWS (CAPA) need to be more broad than the EKS controller because CAPA is responsible for provisioning everything needed for the cluster (VPC, gateway, etc).
+    You don't need to run all of the ACK controllers if all you want is a way to provision an EKS cluster.
+    You can pick and choose which ACK controllers you want to deploy.
+    - With the EKS ACK controller you will get all of the configuration flexibility of the EKS API including things like managed node groups and fargate.
+    This is because the ACK controller is built directly from the EKS API spec and not abstracted to be a general Kubernetes cluster.
+
+## cdk8s
+
+!!! question "Question"
+    How does ACK relate to [cdk8s](https://cdk8s.io/)?
+
+!!! quote "Answer"
+    cdk8s is an open-source software development framework for defining Kubernetes applications and reusable abstractions using familiar programming languages and rich object-oriented APIs.
+    You can use cdk8s to create any resource inside a Kubernetes cluster.
+    This includes CustomResourceDefinition (CRD) resources.
```

Comment:
1) Should be Custom Resources (CRs)

2) Maybe linkify "Custom Resources" to point here: https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/